transform/variant/maf_transform.py

```diff
@@ -0,0 +1,281 @@
+                                        max_workers=workers,
+                                        harvest=harvest,
+                                        filter=filter):
+        # if allele was filtered out
+        if not allele:
+            continue
+        # save the allele that was created
+        emitter.emit_vertex(allele)
+        # create edge between the allele and the callset
+        call_tuples, callsets = _callset_maker(allele, source, centerCol,
+                                               method, line)
+        # save the calls
+        for call_tuple in call_tuples:
+            call = call_tuple[0]
+            callset_gid = call_tuple[1]
+            emitter.emit_edge(call, allele.gid(), callset_gid)
+        # many callsets can be created, emit only uniques
+        for callset in callsets:
+            if callset.gid not in my_callsets_ids:
+                my_callsets_ids.add(callset.gid)
+                emitter.emit_vertex(callset)
+                emitter.emit_edge(CallsetFor(), callset.gid(),
+                                  callset.normal_biosample_id)
+                emitter.emit_edge(CallsetFor(), callset.gid(),
+                                  callset.tumor_biosample_id)
+
+        # create edge to gene
+        ensembl_id = line.get('Gene', None)
+        gene_gid = Gene.make_gid(gene_id=ensembl_id)
+        emitter.emit_edge(AlleleIn(), allele.gid(), gene_gid)
+        # log progress
+        c += 1
+        if c % 1000 == 0:  # pragma nocover
+            logging.info('imported {}'.format(c))
+    logging.info('imported {}'.format(c))
+
+
+def convert(mafpath, prefix, workers=5, skip=0, harvest=True, filter=[]):
+    """ entry point """
+    emitter = Emitter(prefix=prefix)
+    maf_convert(emitter=emitter, mafpath=mafpath, workers=workers, skip=skip,
+                harvest=harvest, filter=filter)
+    emitter.close()
+
+
+if __name__ == '__main__':  # pragma: no cover
+    parser = default_argument_parser()
+    parser.add_argument('--maf_file', type=str,
+                        help='Path to the maf you want to import')
+    parser.add_argument(
+        '--workers', type=int,
+        help="multithread harvest from myvariant.info",
+        default=5
+    )
+    parser.add_argument(
+        '--skip', type=int,
+        help="skip first N lines in MAF",
+        default=0
+    )
+    parser.add_argument('--filter', type=str,
+                        help='Path of already harvested Allele gids')
+    harvest = parser.add_mutually_exclusive_group(required=False)
+    harvest.add_argument('--harvest', dest='harvest',
+                         help="get myvariantinfo", action='store_true')
+    harvest.add_argument('--no-harvest', dest='harvest',
+                         help="do not get myvariantinfo", action='store_false')
+    parser.set_defaults(harvest=True)
+
+    # We don't need the first argument, which is the program name
+    options = parser.parse_args(sys.argv[1:])
```

Comment:
Why do you bother to pass args to parse_args()? It usually gets them automatically.